src/formState.test.tsx

```diff
@@ -1151,9 +1144,11 @@ describe("formState", () => {
 
     // And when the underlying mobx class has a field unset
     instance.firstName = undefined;
-    // Then our fomr-state computed re-runs again
+    // Then our form-state computed re-runs again
     expect(numCalcs).toBe(3);
     expect(formState.fullName.value).toEqual("undefined last");
+    // And knows the field should be sent as null
+    expect(formState.changedValue).toEqual({ firstName: null });
```

Comment:
Before the change, this was showing `firstName: undefined`, which would be dropped from the JSON and not actually sent to the server to unset the first name